shared/src/main/scala/pl/metastack/metatime/Time.scala

```diff
@@ -92,5 +112,13 @@ object Time {
   }
 
   /** Current time */
-  def now(): Time = ???
+     def now() : Time = {
+               val currTimeInMilliSec : Long =   System.currentTimeMillis()
+               new Time{
+                 override val h: Int = (((currTimeInMilliSec/ 1000) / 3600 ) % 24).asInstanceOf[Int]
```

Comment:
Use `toInt` instead. And put a space between operators, but not before brackets.